src/sentry/notifications/activity/base.py

```diff
@@ -15,30 +16,30 @@
 from sentry.utils.linksign import generate_signed_link
 
 
-class ActivityEmail:
-    def __init__(self, activity):
+class ActivityNotification:
+    def __init__(self, activity: Activity) -> None:
         self.activity = activity
         self.project = activity.project
         self.organization = self.project.organization
         self.group = activity.group
 
-    def _get_subject_prefix(self):
+    def _get_subject_prefix(self) -> str:
         prefix = ProjectOption.objects.get_value(project=self.project, key="mail:subject_prefix")
         if not prefix:
             prefix = options.get("mail.subject-prefix")
-        return prefix
+        return str(prefix)
```

Comment:
why does this need to be converted to a string now?

Reply:
`get_value` and `options.get` return `Any`. We know its a string so I'm wrapping it in `str` for the type checker.